src/plugins/data/common/field_formats/converters/boolean.ts

```diff
@@ -17,14 +17,17 @@
  * under the License.
  */
 
+import { i18n } from '@kbn/i18n';
 import { KBN_FIELD_TYPES } from '../../kbn_field_types/types';
 import { FieldFormat } from '../field_format';
 import { TextContextTypeConvert, FIELD_FORMAT_IDS } from '../types';
 import { asPrettyString } from '../utils';
 
 export class BoolFormat extends FieldFormat {
   static id = FIELD_FORMAT_IDS.BOOLEAN;
-  static title = 'Boolean';
+  static title = i18n.translate('data.common.fieldFormats.boolean.title', {
```

Comment:
nit: For our translation IDs, we've been using the structure `${pluginName}.${serviceName}.whatever` as a convention

So ideally this would be `data.fieldFormats.boolean.title`, no need for `common`

Same feedback applies to other IDs in this PR

Reply:
Done